src/ui/home/containers/identity.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { IdentityComponent } from 'src/ui/home/components/identity'
+import { connect } from 'react-redux'
+import { AnyAction } from 'redux'
+import { accountActions } from 'src/actions'
+import { RootState } from 'src/reducers/'
+
+interface ConnectProps {
+  // did: string
+  // setDid: (did: string) => void
+}
+
+interface Props extends ConnectProps {}
+
+interface State {}
+
+
+
+export class IdentityContainer extends React.Component<Props, State> {
+
+  state = {}
+
+  render() {
+    console.log(this.props, 'nav')
+    return <IdentityComponent 
+      // did= { this.props.did }
+    />
+  }
+}
+
+const mapStateToProps = (state: RootState) => {
+  return {
+    // did: state.account.did
+  }
+}
+
+const mapDispatchToProps = (dispatch: (action: AnyAction) => void) => {
```

Comment:
I guess the type of dispatch could be `Dispatch` from redux

Reply:
I can change this when I actually implement the screens. These are just WIP placeholders for now.